package.json

```diff
@@ -74,6 +74,7 @@
   "sideEffects": false,
   "dependencies": {
     "bn.js": "^5.1.2",
+    "cross-fetch": "^3.0.4",
```

Comment:
What was wrong with `node-fetch`? In fact `cross-fetch` just uses `node-fetch` and is intended as an isomorphic library, that is one that works both in the brower and in NodeJS, which is not what we want here since this is used in a Truffle script and not intended to run in the browser.

Reply:
Maybe you were missing the TS type definitions? `yarn add -D @types/node-fetch`

Reply:
> Maybe you were missing the TS type definitions? yarn add -D @types/node-fetch

Yes this is precisely why I switched, I just thought the easiest thing was to switch over. I can also just include the types if this is the preferred choice here. 

Note, though, that this is **not** a dev-dependency, since it is required for the scripts.

Reply:
> Note, though, that this is *not* a dev-dependency, since it is required for the scripts.

The `scripts/` folder is not included in the npm package (not part of the `package.json` `"files"` setting), and are intended to be run from within the repo, and not from an external project that adds `@gnosis.pm/dex-contracts` as a dependency. This means that, currently, `node-fetch` will never be used as a direct dependency because we don't package any file that uses it in the published npm package. So it should stay as a `devDependency` for now.

If I remember correctly, this was done to keep `@gnosis.pm/dex-contracts` as slim as possible for use with in the front-end (that doesn't actually need these scripts).

If we do want to also publish the `scripts/` directory (which I'm not convinced is a good idea as-is), then this should be done in a separate PR, and then a lot more `devDependencies` need to move to `dependencies` (such as `truffle` for example which is required to run truffle scripts!).

Reply:
> I just thought the easiest thing was to switch over. 

I mean, the only downside is some extra browser specific dependencies get pulled in that will never be used, but in the end its not so important.

It should definitely be a `devDependency` either way though.

Reply:
Turns out `whatwg-fetch` is already a transient dependency, so there is no benefit to using `node-fetch` + `@types/node-fetch` over `cross-fetch`. In fact its slightly worse as there are two package versions to track in the package.json instead of just one. So maybe keep using `cross-fetch`, but move it to be a dev-dependency.